compose.yaml

```diff
@@ -178,18 +178,17 @@ services:
     # Needs 80GB of GPU memory. A g6.12xlarge EC2 instance should work.
     # Docs: https://docs.vllm.ai/projects/recipes/en/latest/Llama/Llama4-Scout.html
     extends: common-base
-    image: vllm/vllm-openai:v0.10.0
+    image: vllm/vllm-openai:v0.11.0
```

Comment:
I still couldn't get llama4 to run, but I got closer. With 0.11.0 and downgrading to 4-bit quantization (which is what gpt-oss above uses), I no longer get memory issues. Instead I'm seeing some complaint about the wrong cuda driver version, which might be user-error. But I didn't want to waste too much time on this, so I stopped there. We're closer, but not yet proven to work.